geth/geth-genesis.json

```diff
@@ -15,7 +15,7 @@
     "londonBlock": 0,
     "shardingForkBlock": 66,
     "clique": {
-      "period": 5,
+      "period": 14,
```

Comment:
probably need to update the deposit contract address to `0x4242424242424242424242424242424242424242` under `alloc` as well

Reply:
The geth genesis will be generated by `testnetgen` and we'll no longer need this file.